cla_backend/apps/cla_butler/migrations/0002_auto_20230426_1405.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("cla_butler", "0001_initial")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="diversitydatacheck",
+            name="action",
+            field=models.CharField(max_length=20, choices=[(b"check", b"Check"), (b"reencrypt", b"Re-encrypt")]),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
+            model_name="diversitydatacheck",
+            name="personal_details",
+            field=models.ForeignKey(to="legalaid.PersonalDetails"),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
+            model_name="diversitydatacheck",
+            name="status",
+            field=models.CharField(max_length=10, choices=[(b"ok", b"OK"), (b"fail", b"Fail")]),
```

Comment:
In the initial migration file this did not have the choices property set https://github.com/ministryofjustice/cla_backend/blob/master/cla_backend/apps/cla_butler/migrations/0001_initial.py#L24

I am not sure why it did not originally add the choices property but I don't think have it add them now would cause an issue

Reply:
Awesome, LGTM